lib/config/presets/local/index.ts

```diff
@@ -12,6 +13,7 @@ const resolvers = {
   azure,
   bitbucket,
   'bitbucket-server': bitbucketServer,
+  gerrit,
```

Comment:
```suggestion
  gerrit: local,
```

We only support local presets, as there is no public hosted gerrit like github or gitlab